package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "variable-type",
-  "version": "0.2.1",
+  "version": "1.0.0",
   "description": "Runtime type checking for variable and similar objects.",
-  "main": "src/index.js",
+  "main": "dist/src/index.js",
```

Comment:
this is not good.

Reply:
Agreed. We can switch to lib and update .npmignore but what do you think about this approach? Basically uses make and docker to build/publish and only the lib folder winds up in the package.

https://github.com/ryantate13/variable-type/tree/docker_build

Reply:
github action can do this, no need to use docker.

We just want:

1. tsc build into lib folder
2. perf / test do not build to js